Azure/sonic-buildimage
# Generate interface table to have an entry designated to default VRF.

**- What I did**
Generate interface table to have an entry designated to default VRF. 

Current:
```
    "VLAN_INTERFACE": {
        "Vlan1000|192.168.0.1/21": {}
    },
```

Proposed:
```
    "VLAN_INTERFACE": {
        "Vlan1000": {},
        "Vlan1000|192.168.0.1/21": {}
    },
```

**- How I did it**
Modify minigraph parser:

**- How to verify it**
Run load_minigraph and verify entries without IP prefix in config_db:
```
 2) "LOOPBACK_INTERFACE|Loopback0|FC00:1::32/128"
 3) "MGMT_INTERFACE|eth0|FC00:2::32/64"
 4) "PORTCHANNEL_INTERFACE|PortChannel0002|FC00::75/126"
 5) "PORTCHANNEL_INTERFACE|PortChannel0001|FC00::71/126"
 6) "PORTCHANNEL_INTERFACE|PortChannel0004|10.0.0.62/31"
 7) "PORTCHANNEL_INTERFACE|PortChannel0001"
 8) "PORTCHANNEL_INTERFACE|PortChannel0004|FC00::7D/126"
 9) "PORTCHANNEL_INTERFACE|PortChannel0003|10.0.0.60/31"
10) "PORTCHANNEL_INTERFACE|PortChannel0003"
11) "PORTCHANNEL_INTERFACE|PortChannel0004"
12) "PORTCHANNEL_INTERFACE|PortChannel0003|FC00::79/126"
13) "VLAN_INTERFACE|Vlan1000|192.168.0.1/21"
14) "LOOPBACK_INTERFACE|Loopback0|10.1.0.32/32"
15) "PORTCHANNEL_INTERFACE|PortChannel0001|10.0.0.56/31"
16) "MGMT_INTERFACE|eth0|10.3.147.47/23"
17) "VLAN_INTERFACE|Vlan1000"
18) "PORTCHANNEL_INTERFACE|PortChannel0002|10.0.0.58/31
```


**- Description for the changelog**
This is to handle backward compatibility issues and to address the dependency with https://github.com/Azure/sonic-swss/pull/796


**- A picture of a cute animal (not mandatory but encouraged)**
